app/screens/edit_profile/edit_profile.tsx

```diff
@@ -153,6 +187,18 @@ const EditProfile = ({
                     resetScreenForProfileError(reqError);
                     return;
                 }
+
+                // Update custom attributes if changed
+                if (userInfo.customAttributes) {
+                    logDebug('updating custom profile attributes', userInfo.customAttributes);
+
+                    const values: CustomProfileAttributeSimple = {};
+                    Object.keys(userInfo.customAttributes).forEach((field) => {
+                        values[field] = userInfo.customAttributes[field].value;
+                    });
+                    const client = NetworkManager.getClient(serverUrl);
+                    await client.updateCustomProfileAttributeValues(values);
```

Comment:
We may want to add this to a remote action, to avoid having to deal with the network manager at this level. Also, it will handle the possible thrown exceptions.

Reply:
Also, if there is any error, I wonder if we want to inform the user somehow.